app/presenters/v3/service_instance_presenter.rb

```diff
@@ -9,12 +9,28 @@ def to_hash
             guid:       service_instance.guid,
             created_at: service_instance.created_at,
             updated_at: service_instance.updated_at,
-            name:      service_instance.name
+            name:      service_instance.name,
+            relationships: {
+              space: {
+                data: {
+                  guid: service_instance.space.guid
+                }
+              }
+            },
+            links: {
```

Comment:
To construct `links` in our presenters, we use a `build_links` method, as in [organization_presenter](https://github.com/cloudfoundry/cloud_controller_ng/blob/master/app/presenters/v3/organization_presenter.rb#L19-L25).

We'll merge this PR, but in a follow-up PR could you change this presenter to follow this convention?